### ISSUE TITLE
groupnorm affine failure of TensorRT 8.6.1

### ISSUE THREAD

**Issue by ArtemisZGL** [NVIDIA/TensorRT#3199]
## Description

I used tensorrt python api to convert one model, which has the group normalization(GroupNorm(512, 512, eps=1e-05, affine=True). When I used add_scale_nd for affine operation, it failed with the error below.
```
[TRT] [E] 3: [network.cpp::addScaleNd::1162] Error Code 3: API Usage Error (Parameter check failed at: optimizer/api/network.cpp::addScaleNd::1162, condition: qdqScale || basicScale 
```

the command and its parameter shape are shown below. . It seems the shapes are correct, dont know why this error occured.
```
ctx.network.add_scale_nd(c, trt.ScaleMode.CHANNEL, bias, scale, power, 1)

### shape for each param ###
# c: (1, 512, 13656)
# bias: (512, )
# scale: (512, )
# power: (512, )
```

## Environment

**TensorRT Version**: 8.6.1

**NVIDIA GPU**: 4070

**NVIDIA Driver Version**: 525.125.06

**CUDA Version**: 11.3

**CUDNN Version**:

Operating System: ubuntu 20.04

Python Version (if applicable): 3.10.9

Tensorflow Version (if applicable):

PyTorch Version (if applicable): 1.12.1

Baremetal or Container (if so, version):

## Relevant Files

**Model link**:

## Steps To Reproduce

**Commands or scripts**:

**Have you tried [the latest release](https://developer.nvidia.com/tensorrt)?**:

**Can this model run on other frameworks?** For example run ONNX model with ONNXRuntime (`polygraphy run <model.onnx> --onnxrt`):

**Comment by ArtemisZGL** [comment]
it seems the problem is similar with #1277 and #3094.

**Comment by zerollzeng** [comment]
@ttyio Do you know? Looks like it's related to Q/DQ.

**Comment by zerollzeng** [comment]
I didn't see our api doc mention such limitation, maybe we should improve: https://docs.nvidia.com/deeplearning/tensorrt/api/c_api/classnvinfer1_1_1_i_scale_layer.html

**Comment by ttyio** [comment]
@ArtemisZGL , could you call `add_shuffle` to reshape the `c` into `1, 512, 13656, 1` before `scale_nd`, and reshape it back after `scale_nd`?

**Comment by ttyio** [comment]
BTW, the error here is not precise. We have warning that `Only 2D or 3D scale is supported` in doc: https://docs.nvidia.com/deeplearning/tensorrt/api/c_api/classnvinfer1_1_1_i_network_definition.html#a2386f603fa2e90fd21b9817013e81d3b

**Comment by ArtemisZGL** [comment]
> @ArtemisZGL , could you call `add_shuffle` to reshape the `c` into `1, 512, 13656, 1` before `scale_nd`, and reshape it back after `scale_nd`?

Sure, thanks for reply. I will try it as soon as possible.

**Comment by ttyio** [comment]
Closing since no activity for more than 3 weeks, pls reopen if you still have question, thanks!

**Comment by chaoz-dev** [comment]
Ran into this problem myself as well on TRT 8.5.x. The issue turned out to be that the input tensor needs to be of rank >= 4; the scale can be of rank 1, in my experience. 

The following documentation appears to be correct, and mirrors my experience:
https://docs.nvidia.com/deeplearning/tensorrt/operators/docs/Scale.html
